src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/ImportProcessingJob.cs

```diff
@@ -115,6 +115,12 @@ public async Task<string> ExecuteAsync(JobInfo jobInfo, CancellationToken cancel
                     var error = new ImportProcessingJobErrorResult() { Message = "Due to unauthorized request, import processing operation failed." };
                     throw new JobExecutionException(ex.Message, error, ex);
                 }
+                catch (RequestFailedException ex) when (ex.Status == 404)
```

Comment:
Can we use HttpStatusCode.NotFound instead of 404?

Reply:
Fixed, thanks @abiisnn. 👍🏽